### PR TITLE
Update and rename messaging-basics.md to messaging-basics.rst

### PR DESCRIPTION
**Pull request by lfbrock**

### CODE REVIEW

**Comment by jasonblais** [comment]
We should come up with a guideline on whether to capitalize "Shift", "Alt", and other keyboard buttons in our docs. We're really inconsistent right now.

image

I looked into this previously, and there isn't really a standard as far as I know. We however use capital letters in `shortcuts` and keyboard shortcuts docs (+ in a few other places), so propose using capitals.

Up for discussion though.

File: source/help/getting-started/messaging-basics.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
+Messaging Basics
+================
+
+--------------
+
+**Write messages** using the text input box at the bottom of Mattermost.
+Press **ENTER** to send a message. Use **Shift+ENTER** to create a new
```
Comment (jasonblais) [inline source/help/getting-started/messaging-basics.rst]: We should come up with a guideline on whether to capitalize "Shift", "Alt", and other keyboard buttons in our docs. We're really inconsistent right now.

image

I looked into this previously, and there isn't really a standard as far as I know. We however use capital letters in `shortcuts` and keyboard shortcuts docs (+ in a few other places), so propose using capitals.

Up for discussion though.

Comment (jasonblais) [inline source/help/getting-started/messaging-basics.rst]: Another thing is whether to have spaces between the keyboard button labels, e.g. `SHIFT+ENTER` or `SHIFT + ENTER`.

Again, we're pretty inconsistent.

**Comment by enahum** [comment]
Also changes in the text should be included in platform too

**Comment by lfbrock** [comment]
The only text change was from Shift+Enter to SHIFT+ENTER, but not sure if it's worth updating now in platform because we probably want someone to search for other similar instances in the product help text and make those changes as well.

**Comment by jasonblais** [comment]
@wget Offered to search for other similar instances in the platform repo earlier

**Comment by wget** [comment]
@jasonblais Yep on my todo actually. And wrt. to what we do in others FOSS projects, those have to be uncapitalized

Alt+Shift+H+Enter for example. And these keys have to be translated. Alt+Maj+H+Entrée in French for example.

**Comment by jasonblais** [comment]
Chatted with @wget on the standard in most FOSS projects when it comes to keyboard buttons. I'll bring this up in PM meeting to discuss.

Doesn't block this PR so I'll merge this.

If any changes are needed, I'll add it to our docs style guides, and do a pass across both the platform and docs repos.

**Pull request merged**